src/js/videojs-dash.js

```diff
@@ -70,6 +70,21 @@ class Html5DashJS {
     // element to bind to.
     this.mediaPlayer_.initialize();
 
+    // Retrigger a dash.js-specific error event as a generic HTML5 one
+    // See src/streaming/utils/ErrorHandler.js in dash.js code
+    this.retriggerError = (event) => {
```

Comment:
This should probably get a tail (`_` suffix) to make it pseudoprivate.

Reply:
Yeah, good call.